### ISSUE TITLE
Add steps to build Julia docs

### ISSUE THREAD

**Issue by logankilpatrick** [JuliaLang/julia#43350]
Is there a step by step guide (similar to building Julia itself in the README) but specifically for building the Julia docs?

**Comment by fredrikekre** [comment]
`make docs`

**Comment by logankilpatrick** [comment]
So do I run all the commands here first: https://github.com/JuliaLang/julia#building-julia and then `make docs`?

**Comment by DilumAluthge** [comment]
The full steps to build just the docs are:

```
git clone https://github.com/JuliaLang/julia.git
cd julia
git checkout v1.7.0
make docs
```

**Comment by ViralBShah** [comment]
Yup that's all there is to it. You do need a full latex installation though, and perhaps that should be noted wherever we discuss build dependencies in the manual.

**Comment by ViralBShah** [comment]
We shouldn't add these to the README on github, but in the detailed build docs. Probably should go in here: https://github.com/JuliaLang/julia/tree/master/doc/src/devdocs/build
